### DISCUSSION TITLE
Use squash merge

### DISCUSSION THREAD

**Discussion by ThomasGoulet73** [dotnet/wpf#9875] [General]
I noticed that the latest merged PRs were merged with a merge commit instead of a squash merge like it was previously. I would suggest using squash merge instead of merge commits to have cleaner and more linear git history. FWIW dotnet/runtime always uses squash merge except in specific cases and only repo admins have access to merging using merge commits (See dotnet/runtime#260 for a discussion about this).

/cc @harshit7962 since you were the last person to merge PRs

**Comment by harshit7962** [comment]
@ThomasGoulet73 Thanks for bringing this to our notice and apologies for the oversight. Just checked we were doing it the wrong way, will make sure that we squash and merge from now on.
